internal/v2/s2av2_e2e_test.go

```diff
@@ -126,7 +129,7 @@ func runClient(ctx context.Context, t *testing.T, clientS2AAddress, serverAddr s
 }
 
 func TestEndToEndUsingFakeS2AOverTCP(t *testing.T) {
-	// TODO(rmehta19): Set S2A_ACCESS_TOKEN env variable once #46 merges.
+	os.Setenv(accessTokenEnvVariable, "")
```

Comment:
Why are we setting the env variable to the empty string here? Is the idea that we should have a test where there is no token? If so, I agree, but I think we should call `os.Unsetenv(accessTokenEnvVariable)` instead.

Reply:
I believe I set it to an empty string to replicate the structure of the test in the v1 implementation. I can set a non empty string token here, and create another test with no token using Unsetenv, and make sure the error is being caught.